lib/matplotlib/tests/test_ticker.py

```diff
@@ -159,6 +159,49 @@ def test_SymmetricalLogLocator_set_params():
     nose.tools.assert_equal(sym.numticks, 8)
 
 
+def test_ScalarFormatter_offset_value():
```

Comment:
As it is, the `@cleanup` decorator doesn't support generative tests.  I'll write an patch for that first.